test/test.js

```diff
@@ -5,7 +5,7 @@ const Koa     = require('koa');
 const request = require('supertest');
 const assert  = require('assert');
 const jwt     = require('jsonwebtoken');
-
+const Promise = require('bluebird');
```

Comment:
Node 4 (which is the min specified supported version in our package.json) has native promises, which should be sufficient, rather than introducing a large dependency.